ecl/hql/hqlexpr.cpp

```diff
@@ -14211,7 +14211,7 @@ unsigned exportField(IPropertyTree *table, IHqlExpression *field, unsigned & off
         }
         f->setProp("Expression", ecl.str());
     }
-    f->setPropInt("@rawtype", getClarionResultType(type));
+    f->setPropInt("@rawtype", getRawResultType(type));
```

Comment:
@ghalliday is @rawtype ever used or can this be completely removed?  It ends up being a combination value of field and type following some clarion spec I think.

Reply:
@kenrowland I cannot find this used in any archive.  (I find several hits for ecltype set above.)

I think this should be deleted.